cmds/core/kexec/kexec_linux.go

```diff
@@ -90,9 +93,42 @@ func main() {
 		purgatory.Debug = log.Printf
 	}
 
-	if (!opts.exec && flag.NArg() == 0) || flag.NArg() > 1 {
+	var kernelpath string
+	// Regarding potential conflicting options, e.g.
+	// we are given a kernel path from linux image config, while
+	// also receiving a kernel path from position argument flag.Arg(0)
+	//
+	// Options loaded from linux image config might as well be seen
+	// as default options to begin with, and same option, if specified
+	// via flag, will take precedence and be used for final execution.
```

Comment:
I started rewriting this with what I read at first, then realized what's actually happening, and now this is my suggestion for what I'd write based on what it is right now:

```go
// Load Linux config from file.
// 
// Only the kernel path can be overridden through command line parameters.
```

Writing this -- now that I get it -- I'm gonna suggest we be consistent. Either all options are overridable by flags, or none are and it can only be read from the config. I'd go for none? Unless you really need it?